src/container/srv_target.c

```diff
@@ -1375,7 +1363,7 @@ ds_cont_child_lookup(uuid_t pool_uuid, uuid_t cont_uuid,
 	if (rc != 0)
 		return rc;
 
-	if ((*ds_cont)->sc_stopping) {
+	if ((*ds_cont)->sc_stopping || (*ds_cont)->sc_destroying) {
```

Comment:
[minor] sc_destroying check here is duplicated, cont_child_stop() will set sc_stopping anyway.

Reply:
right, I think we should do more cleanup for this part of code.